functions/Import-DbaCsv.ps1

```diff
@@ -439,7 +439,7 @@ function Import-DbaCsv {
                 $elapsed = [System.Diagnostics.Stopwatch]::StartNew()
                 # Open Connection to SQL Server
                 try {
-                    $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -StatementTimeout 0 -MinimumVersion 9
+                    $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential -Database $Database -StatementTimeout 0 -MinimumVersion 9
```

Comment:
Just add the database parameter to our internal command. We only use the public function on invoke command (there might be a few exclusions).

Reply:
Actually the internal function has the database parameter so please revert this back and include reference to the database parameter.